internal/pkg/tfconf/module.go

```diff
@@ -124,24 +126,35 @@ func CreateModule(options *Options) (*Module, error) {
 		}
 	}
 
```

Comment:
I will change the env variable name if this ends up being an accepted solution - suggestions welcome!

Reply:
Please look at https://github.com/segmentio/terraform-docs/pull/191#discussion_r379701094

Reply:
Removed the env variable, let me know what you think~